new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -753,6 +803,111 @@ public static void scaleAndVerifyCluster(String clusterName,
+        "Persistent volume claim creation failed with ApiException"),
+        "PersistentVolumeClaim creation failed");
+
+    // check the persistent volume and persistent volume claim exist
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for persistent volume {0} exists "
+                    + "(elapsed time {1}ms, remaining time {2}ms)",
+                pvName,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> pvExists(pvName, labelSelector),
+            String.format("pvExists failed with ApiException when checking pv %s", pvName)));
+
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for persistent volume claim {0} exists in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                pvcName,
+                namespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> pvcExists(pvcName, namespace),
+            String.format("pvcExists failed with ApiException when checking pvc %s in namespace %s",
+                pvcName, namespace)));
+  }
+
+  /**
+   * Create ConfigMap containing domain scripts.
+   * @param configMapName name of the ConfigMap to create
+   * @param files files to be added in ConfigMap, including domain creation python script and properties files
+   * @param namespace the namespace in which the ConfigMap to be created
+   * @throws IOException when reading files fails
+   */
+  public static void createConfigMapForDomainCreation(String configMapName,
+                                                      List<Path> files,
+                                                      String namespace) throws IOException {
+
+    // create a ConfigMap of the domain
+    Map<String, String> data = new HashMap<>();
+    for (Path file : files) {
+      data.put(file.getFileName().toString(), readString(file));
+    }
+
+    V1ConfigMap configMap = new V1ConfigMap()
+        .data(data)
+        .metadata(new V1ObjectMeta()
+            .name(configMapName)
+            .namespace(namespace));
+
+    assertTrue(assertDoesNotThrow(() -> createConfigMap(configMap),
+        String.format("createConfigMap failed with ApiException for ConfigMap %s with files %s in namespace %s",
+            configMapName, files, namespace)),
+        String.format("createConfigMap failed while creating ConfigMap %s in namespace %s", configMapName, namespace));
+  }
+
+  /**
+   * Run a job to create a domain in the specified namespace.
+   *
+   * @param jobBody V1Job object to create a domain in the specified namespace
+   * @param namespace the namespace in which the job will be run
+   */
+  public static void runCreateDomainJob(V1Job jobBody, String namespace) {
```

Comment:
The name should be changed for this too, because this is simple create job and wait for it to complete. could be used for any purpose.

Reply:
fixed